### ISSUE TITLE
Training on my own large hand written arabic data set

### ISSUE THREAD

**Issue by bjone** [datalab-to/surya#23]
Is it possible to train on my data set ?

**Comment by VikParuchuri** [comment]
Yes, this model can be finetuned.  I don't have publicly available code for easily doing that, but the model implementation is in this repo.  If you have public data, I'm happy to try to include it in my next training run

**Comment by johnlockejrr** [comment]
@VikParuchuri are you intending in the future to release some code for finetuning? I'm interested in finetuning for Hebrew and Samaritan (RTL like Arabic)

**Comment by tarun-menta** [comment]
Hi @bjone @johnlockejrr 

I know its been a while, but we have an updated model that is a lot better! Better formatting, supports inline and block math using latex, better language support, and a lot more! 

Additionally, we now have a easy-to-use [finetuning setup](https://github.com/datalab-to/surya?tab=readme-ov-file#finetuning-surya-ocr)! Try it out and let us know if you need any help!

**Comment by Konafets** [comment]
@tarun-menta Can you share the script how you created the example dataset? Additional question: Is it better for training using whole pages and with its transcription or just parts of them?
